aws/table_aws_ssm_association.go

```diff
@@ -247,6 +253,58 @@ func getAwsSSMAssociation(ctx context.Context, d *plugin.QueryData, h *plugin.Hy
 	return data.AssociationDescription, nil
 }
 
+func getComplianceItems(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getComplianceItems")
+
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	// get service
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	var instanceIds []*string
+	switch h.Item.(type) {
+	case *ssm.Association:
+		association := *h.Item.(*ssm.Association)
+		targets := association.Targets
+		for _, target := range targets {
+			if *target.Key == "InstanceIds" {
+				instanceIds = target.Values
+			}
+		}
+	case *ssm.AssociationDescription:
+		association := *h.Item.(*ssm.AssociationDescription)
+		targets := association.Targets
+		for _, target := range targets {
+			if *target.Key == "InstanceIds" {
+				instanceIds = target.Values
+			}
+		}
+	}
+
+	// Build the params
+	params := &ssm.ListComplianceItemsInput{
+		ResourceIds: instanceIds,
```

Comment:
Can we have more than one `instanceIds` , if yes then we can only pass on resource Id at a time
```
// The ID for the resources from which to get compliance information. Currently,
// you can only specify one resource ID.
ResourceIds []*string `min:"1" type:"list"
````